.github/workflows/rust.yml

```diff
@@ -81,6 +81,9 @@ jobs:
       - name: Doctests
         run: |
           cargo test --doc --all-features
+      - name: cluster test
+        run: |
+          cargo run --bin sui-cluster-test -- new-local
```

Comment:
We should define a rust test that runs this instead of running it here, that way it'll be run in parallel with the other tests as well as make it so that any dev will transparently run them locally with a `cargo test` or `cargo nextest` invocation

Reply:
Also see https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates specifically:

> CARGO_BIN_EXE_<name> — The absolute path to a binary target's executable. This is only set when building an [integration test](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#integration-tests) or benchmark. This may be used with the [env macro](https://doc.rust-lang.org/std/macro.env.html) to find the executable to run for testing purposes. The <name> is the name of the binary target, exactly as-is. For example, CARGO_BIN_EXE_my-program for a binary named my-program. Binaries are automatically built when the test is built, unless the binary has required features that are not enabled.

If you want to invoke the binary from a test